### PR TITLE
Update print statements to python 3 issue #16

### PR DESCRIPTION
**Pull request by carlostighe**
Update print statement to include `()` so they dont raise exceptions in python 3
Related to issue #16

### CODE REVIEW

File: gdal2tiles.py [hunk 1]
```diff
@@ -612,7 +612,7 @@ def process(self):
     # -------------------------------------------------------------------------
 
     def error(self, msg, details=''):
-        """Print an error message and stop the processing"""
+        """Printan error message and stop the processing"""
```
Comment (commenthol) [inline gdal2tiles.py]: Could you please leave the whitespace?

File: gdal2tiles.py [hunk 2]
```diff
@@ -624,7 +624,7 @@ def error(self, msg, details=''):
     # -------------------------------------------------------------------------
 
     def progressbar(self, complete=0.0):
-        """Print progressbar for float value 0..1"""
+        """Printprogressbar for float value 0..1"""
```
Comment (commenthol) [inline gdal2tiles.py]: And here as well.

**Comment by commenthol** [comment]
Hi @Craicerjack, Thanks for your PR. 
Can you please add back the two spaces and then we are ready to merge.

**Comment by carlostighe** [comment]
Hey @commenthol - fixed those typos (sorry about that, should have checked myself) and changed print statements in multiprocess file too.

**Comment by commenthol** [comment]
Thanks

**Pull request merged**